scanners/sslyze.py

```diff
@@ -14,8 +21,17 @@
 #
 # If data exists for a domain from `pshtt`, will check results
 # and only process domains with valid HTTPS, or broken chains.
+#
+# Supported options:
+#
+# --sslyze-serial - If set, will use a synchronous (single-threaded
+#   in-process) scanner. Defaults to false.
 ###
 
+# Number of seconds to wait during sslyze connection check.
+# Not much patience here, and very willing to move on.
+network_timeout = 5
```

Comment:
This can be overridden via `--network_timeout`, but unless it becomes used/wanted by anyone, I don't see the need to document it.